packages/medusa/src/models/cart.ts

```diff
@@ -396,6 +398,25 @@ export class Cart extends SoftDeletableEntity {
   ])
   sales_channel: SalesChannel
 
+  @FeatureFlagDecorators(
+    [MedusaV2Flag.key, "sales_channels"],
```

Comment:
thought: Should we use the feature flag key for `sales_channels` too?

Reply:
yes, I will refactor